### DISCUSSION TITLE
Confused: Where to start?

### DISCUSSION THREAD

**Discussion by hagenbuch** [benzino77/tasmocompiler#360] [Q&A] [answered]
Hi all,

I am sorry for my noob question. I remember having used the "web compiler" as I would put it before but can by no means find where to click.

- Yes, I do have Github account (I would not be able to write here, right? :-)
- Yes, I somehow seem to have Gitpod access. I am able to "open a workspace" (whatever that is) for building a specific Tasmota version (I would need ESP32 with LAN, with SML for smart metering, that's all)

Then, a lot of terminal messages roll by in my browser, I have seem development environments before but I am very confused here.

What I would expect is a link to click on that brings me to the input mask of the tasmocompiler "website" so I may chose my options. Then I would expect the compilation in my browser to start and then after a while I'd expect a download of the compiled binary. I am not familiar with docker, I am just sitting at a pretty vanilla Linux Mint machine with Firefox.

Glad for any hint. http://localhost:3000 does nothing.

Andreas

**Comment by benzino77** [comment, marked as answer]
Hi Andreas,

I think you have to follow the instructions from [this](https://github.com/benzino77/tasmocompiler#how-to-start-using-tasmocompiler) section of README file.
The tricky part is that most (if not all) of browsers block pop-ups and you probably missed that. All you need to do is to open blocked popup window.

**Reply by hagenbuch** [reply]
Ahhh.. THAT was is, many thanks! Hope this thread will be useful for others :)
